### ISSUE TITLE
Installer fails when installing across partitions

### ISSUE THREAD

**Issue by robhol** [Pathoschild/SMAPI#184]
```
D:\Steam\SteamApps\common\Stardew Valley\SMAPI-1.2-installer>install.exe        
Oops, couldn't find your Stardew Valley install path automatically. You'll need to specify where the game is installed (or install SMAPI manually).                
Enter the game's full directory path (the one containing 'StardewValley.exe' or 'Stardew Valley.exe').
   > D:\Steam\steamapps\common\Stardew Valley                                      OK!
Detected Windows with game in D:\Steam\steamapps\common\Stardew Valley.

You can....
[1] Install SMAPI. You'll need to launch StardewModdingAPI.exe instead afterwards; see the readme.txt for details.
[2] Uninstall SMAPI.
What do you want to do?
1

Copying SMAPI files to game directory...
Adding bundled mods...
   adding TrainerMod...
Found an obsolete mod path: C:\Users\rob\AppData\Roaming\StardewValley\Mods
   Support for mods here was dropped in SMAPI 1.0 (it was never officially supported).
   Moving CJBAutomation into the game's mod directory...

Unhandled Exception: System.IO.IOException: Source and destination path must have identical roots. Move will not work across volumes.
   at System.IO.DirectoryInfo.MoveTo(String destDirName)
   at StardewModdingApi.Installer.InteractiveInstaller.InteractivelyRemoveAppDataMods(Platform platform, DirectoryInfo properModsDir)
   at StardewModdingApi.Installer.InteractiveInstaller.Run(String[] args)
   at StardewModdingApi.Installer.Program.Main(String[] args)

D:\Steam\SteamApps\common\Stardew Valley\SMAPI-1.2-installer>
```

**Comment by Pathoschild** [comment]
Thanks for reporting it; I'll fix that in the upcoming SMAPI 1.4 release.

**Comment by Pathoschild** [comment]
Thanks for reporting it; I'll fix that in the upcoming SMAPI 1.4 release. In the meantime, you can manually move your mods out of `%appdata%\StardewValley\Mods` so the installer doesn't need to perform that step.

**Comment by Pathoschild** [comment]
Fixed in the upcoming 1.4 release.
